### ISSUE TITLE
FileNotFoundError during make run

### ISSUE THREAD

**Issue by bjarman** [BirkbeckCTP/janeway#4015]
**Describe the bug**
Fresh install on a Mac computer. Make install worked fine but make run renders this error:
FileNotFoundError: [Errno 2] No such file or directory: ‘/vol/janeway/src/core/locales/en_us’

**Janeway version**
v.1.5.0 Torres 
455eff2a70ee830dacf0ecf35f29c3f1a2dbd8f6
**To Reproduce**
make install
make run

**Expected behavior**
A complete working install
**Screenshots**

## Front-end Issues
If the issue is front-end specific please add the following details:

**Desktop (please complete the following information):**
 - OS: Mac Os Sonoma

**Smartphone (please complete the following information):**

**Additional context**

**Comment by ajrbyers** [comment]
- [ ] @ajrbyers run docker command to check this does not occur.

**Comment by kelliott121** [comment]
I'm encountering this same problem on my Mac.

**Comment by SimonXIX** [comment]
> > Yes, this is on macOS Tahoe 26.3 (Apple chip) running Docker Desktop. I thought I'd play around locally before getting a Rocky Linux server spun up for proper testing.
> 
> https://github.com/openlibhums/janeway/pull/5047/changes
> 
> Can you take a look at the doc update in this PR? Colleagues from PLOS have submitted it with some updates for MacOS.

Yup, that got it. Changing the Virtual Machine Options as they suggest did the trick. Thanks, Andy!
